doc/release-notes-11413.md

```diff
@@ -0,0 +1,8 @@
+Low-level RPC changes
```

Comment:
Should be "updated or changed RPC", as low level changes are meant for things that are irrelevant to users

Reply:
Switched to "Updated or changed RPC".